functionaltests/src/test/java/util/CustomWebDriver.java

```diff
@@ -14,7 +14,7 @@
     private ChromeDriver driver;
 
     CustomWebDriver() {
-        String currentDriver = SystemUtils.IS_OS_MAC ? "chromedriver_mac_85" : "chromedriver_linux_85";
+        String currentDriver = SystemUtils.IS_OS_MAC ? "chromedriver_mac_84" : "chromedriver_linux_84";
```

Comment:
Is it bacause Travis is using older version?

Reply:
I reverted the broken change that is added by Andi he added v85 which is beta, travis chrome addon has 2 options beta and stable, stable is 84.

in desc I said: `chromedriver was in 79 and the stable version is 84, so it's updated.`

Reply:
chromedriver 84 is the latest stable version that chrome is using. 85 is beta release

Reply:
for more details: https://docs.travis-ci.com/user/chrome#selecting-a-chrome-version

Reply:
clear. Thanks